types/sharp/index.d.ts

```diff
@@ -727,6 +733,8 @@ declare namespace sharp {
          * An integral Number of pixels, zero or false to remove limit, true to use default limit of 268402689 (0x3FFF x 0x3FFF). (optional, default 268402689)
          */
         limitInputPixels?: number | boolean | undefined;
+        /** Set this to true to remove safety features that help prevent memory exhaustion (SVG, PNG). (optional, default false) */
+        unlimited?: boolean | undefined;
```

Comment:
Optional params are union typed with `undefined` implictly \[1], so we could drop the `|  undefined` on all the optional params I think.

\[1]: https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters

Reply:
Yeah, I figured I would just keep things consistent with the current style, but will consider doing another PR to remove the superfluous `| undefined`  on all of these